.github/workflows/ci_reusable.yaml

```diff
@@ -174,3 +179,39 @@ jobs:
       - name: Get the output url
         if: ${{ inputs.DEPLOY_TO_FLEEK }}
         run: echo "Deploy url is ${{ steps.deploy.outputs.deployUrl }}"
+      - name: Build dapp for the preview
+        run: npm run build
+        if: ${{ inputs.IS_PREVIEW == true && inputs.DEPLOY_TO_FLEEK }}
```

Comment:
these conditions are never met (when PREVIEW is true, DEPLOY_TO_FLEEK is false)
Is it a provision for future cases?
Otherwise I think the PREVIEW flag is worthless

Reply:
true, removed step